api_app/analyzers_manager/views.py

```diff
@@ -70,3 +70,11 @@ def get_object(self, job_id, analyzer_name) -> AnalyzerReport:
     def _post_kill(self, report):
         # clean up job
         analyzers_controller.job_cleanup(report.job)
+
+    def _start_retry(self, report: AnalyzerReport):
+        analyzers_to_execute = [report.analyzer_name]
+        report.job.status = "running"  # update job status to running
```

Comment:
shortway would be to use the `Job.update_status()` method.

Reply:
Actually, since we are calling the `start_analyzers`, this is not required.

See here: https://github.com/intelowlproject/IntelOwl/blob/03b51e47269ad4af27e0c3c09338ea8f0ce2ea63/api_app/analyzers_manager/controller.py#L123-L125